### PR TITLE
switch to pip install

### PR DESCRIPTION
**Pull request by weaverba137**
Checklist
* [X] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [X] Bumped the build number (if the version is unchanged)
* [X] Reset the build number to `0` (if the version changed)
* [X] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [X] Ensured the license file is being packaged.

Switching to `pip install` instead of `python setup.py install`.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -13,16 +13,16 @@ source:
   '{{ hash_type }}': '{{ hash_value }}'
 
 build:
-  number: 1
+  number: 2
```
Comment (bsipocz) [inline recipe/meta.yaml:16]: I don't think you need to rebuild this to do the swicth, there should be no difference from the user pov

**Comment by weaverba137** [comment]
Ever seen this?
```
2020-02-19T21:37:53.0820785Z Error: Failed to render jinja template in /home/conda/recipe_root/meta.yaml:
2020-02-19T21:37:53.0821890Z 'PYTHON' is undefined
```
I copied the recipe directly from https://conda-forge.org/docs/maintainer/adding_pkgs.html#use-pip

File: recipe/meta.yaml [hunk 2]
```diff
@@ -13,16 +13,16 @@ source:
...
   entry_points:
     - get_juldate = pydl.goddard.astro:get_juldate_main
     - hogg_iau_name = pydl.pydlutils.misc:hogg_iau_name_main
     - compute_templates = pydl.pydlspec2d.spec1d:template_input_main
-  script: python setup.py install --offline --single-version-externally-managed --record=record.txt
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
```
Comment (bsipocz) [inline recipe/meta.yaml]:
```suggestion
  script: {{ PYTHON }} -m pip install . --no-deps -vv
```

Comment (weaverba137) [reply recipe/meta.yaml]: That made no difference.

File: recipe/meta.yaml (hunk 1 above)
Comment (weaverba137) [reply recipe/meta.yaml:16]: Thanx, but I'll leave that as "better safe than sorry".

File: recipe/meta.yaml (hunk 2 above)
Comment (weaverba137) [reply recipe/meta.yaml]: Ah, see https://github.com/conda-forge/conda-smithy/issues/1182

**Comment by weaverba137** [comment]
@bsipocz , can you recommend a template feedstock that uses pip?  This does not appear to be viable as a one-line change.

**Comment by weaverba137** [comment]
@bsipocz , I looked at astroquery-feedstock, and noticed this comment in meta.yaml:
```
# Note: pip-based install fails due to wanting to download astropy-helpers
```
So I guess this is an unsolved problem?

**Comment by weaverba137** [comment]
I will hold off on switching to pip install until I release a version that doesn't require `astropy-helpers`.

**Pull request closed without merging**